docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -239,21 +272,21 @@ The *Run Details* page includes detailed status and duration metadata about the
 
 Learn more about the [Dagger Cloud user interface](./reference/741031-user-interface.mdx).
 
-## Step 4: Integrate the Dagger Cloud cache
+## Step 4: Use cache volumes with the experimental Dagger Cloud cache
 
 :::info
-At the end of this step, you will have integrated Dagger Cloud's distributed cache with your CI pipeline.
+At the end of this step, you will have integrated Dagger Cloud's experimental distributed cache with your workflows.
 :::
 
-:::note
+:::info
 Dagger Cloud's distributed caching feature is only available under the *Team* plan.
 :::
 
 One of Dagger's most powerful features is its ability to [cache data across pipeline runs](../quickstart/635927-caching.mdx). This is especially useful when dealing with package managers such as `npm`, `maven`, `pip` and similar. For these tools to cache properly, they need their own cache data (usually a directory) to be persisted between runs. Since these dependencies are usually locked to specific versions in the application's manifest, re-downloading them on every pipeline run is inefficient and time-consuming.
 
 Dagger comes with built-in support to define one or more such directories as cache volumes and persist their contents across runs. Dagger Cloud enhances caching support significantly and allows cross-host synchronization of cache volumes. This allows multiple machines, including ephemeral runners, to intelligently share a distributed cache.
 
-Dagger Cloud automatically detects and creates cache volumes when they are declared in your code. To see how this works, add a cache volume to your Dagger pipeline and then trigger a CI run. If you're using the starter application and Dagger pipeline from [Appendix A](#appendix-a-create-a-dagger-pipeline), do this by updating the Dagger pipeline code as shown below (changes are highlighted):
+Dagger Cloud automatically detects and creates cache volumes when they are declared in your code. To see how this works, add a cache volume to your Dagger pipeline and then trigger a run. If you're using the starter application and Dagger pipeline from [Appendix A](#appendix-a-create-a-dagger-pipeline), do this by updating the Dagger pipeline code as shown below (changes are highlighted):
```

Comment:
I think we should revisit this paragraph in general as it puts too much emphasis on cache volumes vs caching operations. Let's put that in the queue @vikram-dagger

Reply:
I didn't understand this @marcosnils . What change are you suggesting? That we should not mention cache volumes here and only focus on the experimental cache?